js/classes/entities/midground/drops/rune_drop.js

```diff
@@ -72,7 +72,7 @@ class RuneDrop {
 
         const text = "$" + this.amount;
         const textWidth = CTX.measureText(text).width;
-        const worldPos = Vector.worldToCanvasSpace(this.pos);
+        const worldPos = Vector.worldToCanvasSpace(Vector.add(this.pos, new Vector(0, -ItemLabel.TEXT_SIZE)));
```

Comment:
It may be good to create a new constant within the `RuneDrop` class for its label's text size. Tying it to `ItemLabel`'s text size may cause problems if `ItemLabel`'s text size is updated later to a value that doesn't work well with `RuneDrop`.